unified-codes/apps/web/stories/EntityBrowser.stories.tsx

```diff
@@ -1,13 +1,17 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 import * as React from "react";
 import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
-import { EntityNode, Entity } from "../src/types";
+import { EntityNode, Entity } from "@unified-codes/data";
+
 import {
   Alert,
-  AlertProps,
-  EntityBrowser,
   Severity,
   Snackbar,
-} from "../src/components";
+} from "@unified-codes/ui";
+
+import {
+  EntityBrowser
+} from "@unified-codes/feature";
```

Comment:
this is the other `feature` import; which I think needs updating.

Reply:
a8cd7e3ee1fec42148423fae1f81cebda196c312.